packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -530,10 +538,17 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   }
 
   get citationFields() {
-    return this.fieldsToIncludeInCitations
-      ?.split(',')
-      .map((field) => field.trim())
-      .filter((field) => field.length > 0);
+    const userCitationFields =
+      this.fieldsToIncludeInCitations
+        ?.split(',')
+        .map((field) => field.trim())
+        .filter((field) => field.length > 0) || [];
+
+    const combinedCitationFields = [
+      ...DEFAULT_CITATION_FIELDS,
+      ...userCitationFields,
+    ];
+    return [...new Set(combinedCitationFields)];
```

Comment:
```suggestion
return [...new Set([...this.fieldsToIncludeInCitations?.split(','), DEFAULT_CITATION_FIELDS.split(',')].map((field) => field.trim()).filter((field) => field.length > 0))];
```

This works if you want lol

Reply:
yeah this is not super readable, ill leave it as is